main.go

```diff
@@ -415,7 +415,8 @@ This tool should be ran from the root of the project repository for a new releas
 		r.Dependencies = updatedDeps
 		if highlights {
 			r.Highlights = groupHighlights(projectChanges)
-		} else {
+		}
+		if !highlights || !skipCommits {
```

Comment:
This will cause the commits to be displayed even if the `--skip-commits` is set to true, but highlights flag is not set, it defaults to false.

Reply:
Yep. Doc definitely helps.